### PR TITLE
small changes to about,index, and main.css

### PR DESCRIPTION
**Pull request by aylinev2**
- made small changes to about and index pages. (my grammar isn't the best so if u see any typos pls point them out if you'd like)
- when uploading the site to appengine some stuff wasn't working right so I played around with one thing in main.css and it fixed all the weird errors

### CODE REVIEW

**Review by abeltran1804 — APPROVED** [review]
all looks good!

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 1]
```diff
@@ -31,16 +31,14 @@ DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDateTime(FormatStyle.
   <meta charset="UTF-8">
   <title><%= user.getName() %>'s Profile Page</title>
   <link rel="stylesheet" href="/css/main.css">
-
-  <script>
+</head>
+<script>
```
Comment (acolwell) [inline src/main/webapp/WEB-INF/view/profile.jsp:35]: Why did you move this out of the <head>? I was under the impression that a <script> always needed to be within a <head> or a <body>. It looks like it is in neither now.

Comment (aylinev2) [reply src/main/webapp/WEB-INF/view/profile.jsp:35]: for some reason the first time I updated the appengine, to see if all was working, the scroll function didn't want to work so I moved it around and it was working and I just changed it back right now and it works again? so i'm a bit confused lol

Comment (acolwell) [reply src/main/webapp/WEB-INF/view/profile.jsp:35]: Heh.. I thought that might be the case. I definitely recommend keeping it inside the <head> if you can. I'm not exactly sure what might have been going wrong before.
